app/models/manageiq/providers/ansible_tower/inventory/persister/definitions/automation_collections.rb

```diff
@@ -32,14 +34,4 @@ def add_configuration_script_payloads(extra_properties = {})
       )
     end
   end
-
-  def add_vms
```

Comment:
where was `add_vms` being called from?  don't seem to see that caller being deleted in this PR

Reply:
It is [[here]](https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/272/files#diff-4949e15a250d6715677f53bc2c31a7e36a634a7b85529019198fe64ec1b78574R20) on line 20 just forgot to remove it thanks